### PR TITLE
Line-endings and Git fixing

### PR DESCRIPTION
**Pull request by mortend**
Normalize line-endings and fix-up Git config files.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```
Comment (kusma) [inline .gitattributes]: Uuuuuh, what? This forces everyone to use LF-newlines on all platforms, which isn't a good idea IMO.

**Review by kusma — CHANGES_REQUESTED** [review]
The rest seems fine, but forcing LF-newlines on Windows isn't a great idea. It leads to trouble when people use unexpected tools, and text-editors is something that there's a lot of opinions about out there. Better stick with the platform standards.

File: .gitattributes (hunk 1 above)
Comment (mortend) [reply .gitattributes]: I'm editing a few files but `git diff` on Windows shows me changes on all newlines in a bunch of files. I added `.gitattributes` to make `git diff` useful, but that `eol=lf` part I added is probably overkill. Thanks for pointing that out, I'll remove that part again.

**Review by kusma — APPROVED** [review]
Nice!

**Pull request merged**